### ISSUE TITLE
Evaluate changes to Privacy page for CHS merger

### ISSUE THREAD

**Issue by mekline** [lookit/lookit-api#1181]
# TL;DR

Privacy (https://lookit.mit.edu/privacy/)
    `web/templates/web/privacy.html`
    Same as with the FAQ page, this one has lots of references to Lookit as the site and/or platform. 

# Narrative
What is the user story for this feature? the format should be as follows, e.g.:  "As a Researcher, I would like to have a new view to see all consented videos for a given experiment."

# Acceptance Criteria
A precise description of how/when this issue will be considered closed, e.g.:

- [ ] Tests for new feature
- [ ] QA on new feature
- [ ] Sign off from Kim

# Implementation Notes
Leave important notes/suggestions here for implementation (important files, suggested approaches, etc.)
